### ISSUE TITLE
[Package Request] Proxmox Backup Client

### ISSUE THREAD

**Issue by edersong** [SynoCommunity/spksrc#6308]
### Software Name

Proxmox Backup Client

### Brief Description

Proxmox Backup Client is a client backup tool to perform file backup to Proxmox Backup Server.
As there is no original binary for arm technology, proxmoxbackupclient_go can be used to be able to perform file backups directly from Synology 
Ref.: https://github.com/tizbac/proxmoxbackupclient_go/releases 

### Website

https://github.com/tizbac/proxmoxbackupclient_go/

### Documentation

https://github.com/tizbac/proxmoxbackupclient_go

### Build/Installation documentation

https://github.com/tizbac/proxmoxbackupclient_go

### Source code

https://github.com/tizbac/proxmoxbackupclient_go

### OpenSource License

GPLv3

**Comment by hgy59** [comment]
@edersong what do you expect? It seems that `proxmoxbackupclient_go` is a tool for desktop systems and not for headless linux.

`proxmoxbackupclient_go` depends on gtk, that does not work on headless systems.
But we might try the original `proxmox-backup-client`.

**Comment by hgy59** [comment]
> The native backup client itself would be the better take, I a gree.

no, the original seems even worse to build (rust with custom crates repo).

**Comment by bennetgallein** [comment]
> > The native backup client itself would be the better take, I a gree.
> 
> no, the original seems even worse to build (rust with custom crates repo).

I did not say it's easier to build, but it would be better to use the official binary instead of a third-party one.

**Comment by bennetgallein** [comment]
There has been a release for a statically compiled client:
https://pbs.proxmox.com/docs/installation.html#install-statically-linked-proxmox-backup-client

it can be extracted from this archive: http://download.proxmox.com/debian/pbs-client/dists/bookworm/main/binary-amd64/proxmox-backup-client-static_3.4.0-1_amd64.deb

**Comment by hgy59** [comment]
proxmox-backup-client installation instructions (in german)
https://www.bachmann-lan.de/proxmox-backup-client-xpenology/


EDIT:
debian packages are available for x64 (amd64) only.
The static version runs without other libraries.

@bennetgallein do you still need it for arm archs?

**Comment by bennetgallein** [comment]
https://bugzilla.proxmox.com/show_bug.cgi?id=4788

I swear I read somewhere about an ARM build, but I cannot find it again.

I requested this on behalf of the community. I run a commercial project that (I think) would benefit from having this.

**Comment by hgy59** [comment]
Found a solution to compile proxmoxbackupclient_go for linux (see #6628).
We welcome any feedback.

**Comment by bennetgallein** [comment]
This is a great first step to get this running!

In order to make it more user-friendly I was thinking about a configuration-page in the Webapp where you can enter the hostname and user-name password and perhaps select the volumes you want to backup.

I don't know about GUI development so I cannot make an estimate how much work that would be.
